### ISSUE TITLE
A+=1 compile result is A = A + True 

### ISSUE THREAD

**Issue by 2020snc** [Zezombye/overpy#201]
It doesn't cause any problems but looks odd.

overpy script
```python
 
#!disableOptimizations

rule "test":
    A += 1

```
complile result:
```
variables {
    global:
        0: A
}
rule ("test") {
    event {
        Ongoing - Global;
    }
    actions {
        Set Global Variable(A, Add(Global.A, True));
    }
}

```

**Comment by Zezombye** [comment]
Indeed, the 1->True replacement shouldn't be done if optimization is disabled.

For info, these replacements are done to save elements, as True is one less element than the number 1.

**Comment by 2020snc** [comment]
> Indeed, the 1->True replacement shouldn't be done if optimization is disabled.
> 
> For info, these replacements are done to save elements, as True is one less element than the number 1.

I see, thank you for your thoughtful consideration of optimization.
I did disable the optimization and it still made 1 -> True, overpy version is 6.1.0.
